common/lib/xmodule/xmodule/services.py

```diff
@@ -95,3 +96,26 @@ def __init__(self, configuration_model):
             )
 
         self.configuration = configuration_model
+
+
+class TeamsConfigurationService(object):
+    """
+    An XBlock service that returns the teams_configuration object for a course.
+    """
+    def __init__(self):
+        self._course = None
+
+    def get_course(self, course_id):
+        """
+        Return the course instance associated with this TeamsConfigurationService.
+        This default implementation looks up the course from the modulestore.
+        """
+        return modulestore().get_course(course_id)
+
+    def get_teams_configuration(self, course_id):
+        """
+        Returns the team configuration for a given course.id
+        """
+        if not self._course:
+            self._course = modulestore().get_course(course_id)
```

Comment:
You'd actually want this to be `self._course = self.get_course()`.

Reply:
Corrected.